tests/configurable-timeout-test.ts

```diff
@@ -36,11 +36,11 @@ QUnit.test('We can use a custom later', function(assert) {
     }
   });
 
-  bb.scheduleOnce('one', () => {
+  bb.later(() => {
```

Comment:
I think it would be better to keep the test as before, and fallback `next` and `clearNext` to passed `_platform.setTimeout` and `_platform.clearTimeout` ?